safeclib/strcasecmp_s.c

```diff
@@ -32,7 +32,7 @@
 #include "safeclib_private.h"
 #include "safe_str_constraint.h"
 #include "safe_str_lib.h"
-
+#include <ctype.h>
```

Comment:
safeclib_private.h has it defined, can you re-check? Or you missed STDC_HEADERS somehow